### PR TITLE
Added subtypes to energy cards

### PR DESCRIPTION
**Pull request by Cealgair**
The cards swsh11-171, swsh12-168, swsh12-169, swsh12-215 were missing that subtype, so I added it

### CODE REVIEW

**Comment by Cealgair** [comment]
I noticed there were a bunch of basic energy cards without the "Basic" subtype, so I added them to this PR

File: cards/en/swsh12pt5.json [hunk 1]
```diff
@@ -8279,7 +8282,10 @@
     "id": "swsh12pt5-153",
     "name": "Fire Energy",
     "supertype": "Energy",
-    "number": "153",
+    "subtypes": [
+      "Basic"
+    ],
+    "number": "152",], "153",
```
Comment (adback03) [inline cards/en/swsh12pt5.json:8288]: Looks like some mistakes copy/paste errors perhaps appeared throughout the file.

**Review by adback03 — CHANGES_REQUESTED** [review]
Just need to clean up the numbers that have `,], "235",` and others like that in swsh7.json and swsh12pt5.json

File: cards/en/swsh12pt5.json (hunk 1 above)
Comment (Cealgair) [reply cards/en/swsh12pt5.json]: I think I fixed everything, thanks

**Comment by Cealgair** [comment]
This will close #397

**Pull request merged**